README.md

```diff
@@ -46,29 +46,33 @@ Shopify’s ESLint configs come bundled in this package. In order to use them, y
 }
 ```
 
-If using React, extend the React version of the configuration (which adds some React-specific rules to those in the ESNext config):
+If you are working on an ES5 project, extend the ES5 version of the configuration:
 
 ```json
 {
-  "extends": "plugin:shopify/react"
+  "extends": "plugin:shopify/es5"
 }
 ```
 
-If working on an ES5 project, extend the ES5 version of the configuration:
+You can also add some "augmenting" configs on top of the "core" config by extending an array of linting configs. For example, the following configuration would provide a base ESNext config that is augmented by a jest config:
 
 ```json
 {
-  "extends": "plugin:shopify/es5"
+  "extends": [
+    "plugin:shopify/esnext",
+    "plugin:shopify/jest"
```

Comment:
I'd rather we use another config that should apply universally (IMO most projects should have the jest rule isolated to an `override`)